api/staleness_query.py

```diff
@@ -27,7 +27,8 @@ def get_staleness_obj(identity=None):
 
 
 def get_sys_default_staleness(config=None):
```

Comment:
If you receive the org_id as parameter you should be able to workaround the `connexion` context issue in the unit tests

```suggestion
def get_sys_default_staleness(org_id, config=None):
```